src/scripts.js

```diff
@@ -76,9 +92,9 @@ function fetchAllData() {
     fetchData("ingredients"),
     fetchData("recipes"),
   ]).then((data) => {
-    userData = data[0].usersData;
-    ingredData = data[1].ingredientsData;
-    recipeData = data[2].recipeData;
+    userData = data[0]
```

Comment:
Good catch on the different datatypes coming from the local server as opposed to the initial api we fetched